docker/docker-compose.yml

```diff
@@ -277,6 +277,19 @@ services:
       - "manager-config:/var/lib/chroma"
     environment:
       - RUST_LOG=info
+  iml-mailbox:
+    image: "imlteam/iml-mailbox:dev"
+    hostname: "iml-mailbox"
+    build:
+      context: ../
+      dockerfile: ./docker/iml-mailbox.dockerfile
+    deploy: *default-deploy
+    volumes:
+      - "mailbox:/var/spool/iml/mailbox"
```

Comment:
Will these files ever age out? In RPM based installs I think they are removed after 30 days.

Reply:
Looking into this.

Reply:
I don't see anything specific on aging files out on volumes. How is this done on the rpm version?

Reply:
https://github.com/whamcloud/integrated-manager-for-lustre/blob/ba3fd1aa6328431486367bdfc8451a1348b57656/iml-mailbox.conf

Reply:
Now that this is using systemd I ran a test to verify that the file would age out and confirmed that it was removed. Note that I set the Age out value to be 0 so that as soon as the systemd-tmpfiles-clean.timer executed the file would be removed.

Reply:
How often does `systemd-tmpfiles-clean.timer` fire? Do we need to persist any metadata about it outside the container?